e2e/tests/core/03-connection/connection_test.go

```diff
@@ -33,8 +33,8 @@ type ConnectionTestSuite struct {
 	testsuite.E2ETestSuite
 }
 
-func (s *ConnectionTestSuite) SetupTest() {
-	s.SetupPaths(ibc.DefaultClientOpts(), s.TransferChannelOptions())
+func (s *ConnectionTestSuite) SetupConnectionTestPath(testName string) (ibc.Relayer, ibc.ChannelOutput) {
```

Comment:
it's no longer a lifecycle method, but one we call at the start of tests. the same thing applies for every change of `SetupTest`